deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -26,50 +24,77 @@ export default migration('1686953660_configurate_and_ens', {
     const ethers = deploymentManager.hre.ethers;
     const { utils } = ethers;
 
+    const cometFactory = await deploymentManager.fromDep('cometFactory', 'arbitrum', 'usdc');
     const {
       bridgeReceiver,
       timelock: l2Timelock,
       comet,
       cometAdmin,
       configurator,
       rewards,
+      bridgedComet,
     } = await deploymentManager.getContracts();
 
     const {
       arbitrumInbox,
       arbitrumL1GatewayRouter,
       timelock,
-      comptrollerV2,
       governor,
       USDC,
-      COMP,
+      mainnetCCTPTokenMessenger,
     } = await govDeploymentManager.getContracts();
 
     const refundAddress = l2Timelock.address;
     const configuration = await getConfigurationStruct(deploymentManager);
-
+    const setFactoryCalldata = await calldata(
+      configurator.populateTransaction.setFactory(comet.address, cometFactory.address)
+    );
     const setConfigurationCalldata = await calldata(
       configurator.populateTransaction.setConfiguration(comet.address, configuration)
     );
+    const USDCAmountToBridge = exp(10_000, 6);
+    // CCTP destination domain for Arbitrum
+    const ArbitrumDestinationDomain = 3;
     const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(
       ['address', 'address'],
       [configurator.address, comet.address]
     );
+    
     const setRewardConfigCalldata = utils.defaultAbiCoder.encode(
       ['address', 'address'],
       [comet.address, arbitrumCOMPAddress]
     );
+
+    const turnOffBridgeCometSupplySpeedCalldata = utils.defaultAbiCoder.encode(
```

Comment:
Could we call it `BridgedUSDCComet` or `USDCEComet`?

Reply:
Ok I will now on call it as usdceComet :) 
Will go ahead and rename all instances to be usdcecomet 👍